### ISSUE TITLE
Mime type is not determined on an SVG file that contains <?xml> info tag. 

### ISSUE THREAD

**Issue by nkorobkov** [imgurbot12/pypub#19]
### Problem

When rendering chapters that contain svg images that have <?xml> tag mime type fails to be recognized and image is not loaded. 

Example image: https://res.cloudinary.com/lesswrong-2-0/image/upload/v1561065393/sequences/Argument%20Screens%20Off%20Authority/ArgumentScreensOffAuthority_diagram_4.svg

When you curl for it, response starts with 
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
```

while the code expects it to start with `<svg` https://github.com/imgurbot12/pypub/blob/5e5cd1ffc8c0467cec13ceaafc0ace197609849a/pypub/factory.py#L103

### Steps to reproduce

try building a book from this url: https://www.lesswrong.com/s/6xgy8XYEisLk3tCjH/p/5yFRd3cjLpm3Nd6Di
(contains example picture from above)

### Possible solutions

We can remove the check and just rely on the url ending with .svg or replace `startsWith` with `contains` to check for SVG tag anywhere in the first chunk of data, I assume that should be sufficient.

**Comment by nkorobkov** [comment]
@imgurbot12 
If we agree on the approach, I'm happy to submit PR

**Comment by imgurbot12** [comment]
@nkorobkov thanks for taking the time to submit an issue. always happy to see others using the library lol.

yeah I like the `contains` approach, I think having a basic content check helps avoid any issues with bad or potentially broken images/data so that should work nicely.

Would certainly welcome the PR if you're willing, please and thank you :)

**Comment by imgurbot12** [comment]
@nkorobkov changes have been published as `2.0.5` on pypi. thanks for your submission and support :)
